ksa_compliance/output_models/e_invoice_output_model.py

```diff
@@ -727,7 +730,7 @@ def validate_scheme_with_order(self, field_value: dict, ordered_list: list):
                     f"Invalid scheme ID Order: "
                     f"for {field_value} in Additional IDs")
                 return False
-            elif value is not None:
+            elif value is not None and str(value).strip() != "":
```

Comment:
It's better to modify the value initialization/assignment itself in line 724, something along the lines of:

```python
value = value.get('value').strip() or None
```
We shouldn't be reassigning to the loop variable `value` either, we should use a new variable inside the loop to denote the value.